zproject/computed_settings.py

```diff
@@ -315,6 +315,8 @@ def get_dirs(self) -> List[PosixPath]:
 
 POSTGRESQL_MISSING_DICTIONARIES = bool(get_config("postgresql", "missing_dictionaries", None))
 
+DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
```

Comment:
I wonder if it's a good idea to adjust all of our existing database table to set their autofield explicitly and then set to this BigAutoField?  I don't think we're likely to add new giant tables in the future, but it'd be a nice way to mark the fact that we'd prefer to have been on the Django 3.2 default, and it'll slightly reduce work for a future overall migration.

(Definitely not something we need to do atomically with merging this).

Reply:
Yeah, that's probably good to do in a follow-up commit